content/webapp/components/ThemeSourcedDescription/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { FunctionComponent, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import Space from '@weco/common/views/components/styled/Space';
+import { font } from '@weco/common/utils/classnames';
+import WikidataLogo from '@weco/content/components/ThemeSourcedDescription/WikidataLogo';
+import MeshLogo from '@weco/content/components/ThemeSourcedDescription/mesh-logo.png';
+import Image from 'next/image';
+
+const SOURCE_BOX_WIDTH = 224;
+
+const SourcePill = styled(Space).attrs({
+  className: `${font('intr', 6)} source-pill`,
+})`
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  border-radius: 20px;
+  background-color: ${props => props.theme.color('accent.green')}40;
+  cursor: default;
+  height: 22px;
+  vertical-align: middle;
+
+  &:is(:hover, :focus-within) .source-box-container {
+    opacity: 1;
+    visibility: visible;
+  }
+`;
+
+const Paragraph = styled(Space).attrs({
```

Comment:
I don't think this needs to be `Space` if we're not using the `Space` spacing params, could it be a `p` outright?